src/types/mod.rs

```diff
@@ -14,7 +14,7 @@ mod sexp;
 mod string;
 mod r#struct;
 pub(crate) mod symbol;
-mod timestamp;
+pub(crate) mod timestamp;
```

Comment:
Is this needed so that you can use `TimestampBuilder` in a function signature? If so, I wonder if we should actually change L29 to include `TimestampBuilder`.